content/en/containers/guide/podman-support-with-docker-integration.md

```diff
@@ -14,23 +14,53 @@ The Datadog Agent works with both rootless and rootful containers.
 ## Requirements
 
 * Podman version >= 3.2.0
-* Datadog Agent version >= 7.30.0
+* Datadog Agent version >= 7.54.0
+
+## Host Agent installation
+
+Host Agent installation requires quite a few manual steps and permission tweaks for the Agent to be able to collect the data. Those fixes are not always permanent and are required again if containers restart.
+We advice to follow `rootles` and `rootful` installation instead since it is easier to perform and maintain.
 
 ## Agent deployment as a Podman rootless container
 
 To deploy the Agent as a rootless Podman container, the command to run is similar to the one used for [Docker][2].
 
-The main difference is that as the Agent does not have access to the runtime socket, it relies on the Podman DB to extract the container information that it needs. Instead of mounting the Docker socket and setting `DOCKER_HOST`, you need to mount the Podman DB (`PODMAN_DB_PATH>` in the command below).
+The main difference is that as the Agent does not need to have access to the runtime socket, it relies on the Podman DB to extract the container information that it needs. Instead of mounting the Docker socket and setting `DOCKER_HOST`, the only thing needed is to mount the Podman DB location into the container.
+
+Setting up Podman DB path depends on the Agent version.
+
+**Agent < v7.54.0**
+User needs to set the path to Podman DB (`<PODMAN_DB_PATH>` in the command below).
+
+To discover the exact location of Podman DB, run the following command:
+```
+$ podman info -f json | jq '.store.graphRoot'
+"$HOME/.local/share/containers/storage"
+```
+Check which DB you have at that path and set `<PODMAN_DB_PATH>` accordingly.
```

Comment:
~~~suggestion
1. Set the database at that path to the environment variable `<PODMAN_DB_PATH>`. Replace `path/to/db` with the path to the database:
   ```shell
   export PODMAN_DB_PATH=path/to/db
   ```
~~~
It's useful to give people the command to speed things up. Is a temporary mapping like this okay?